src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -84,6 +84,17 @@ Copyright (c) .NET Foundation. All rights reserved.
     <UserProfileRuntimeStorePath Condition="'$(UserProfileRuntimeStorePath)' == ''">$(_DefaultUserProfileRuntimeStorePath)</UserProfileRuntimeStorePath>
   </PropertyGroup>
 
+  <!-- Opt into .NET Core resource-serialization strategy by default when targeting frameworks
+       that support it by default.
+       -->
+  <PropertyGroup>
+    <GenerateResourceUsePreserializedResources
+      Condition="'$(GenerateResourceUsePreserializedResources)' == '' and
+                 ( ('$(TargetFrameworkIdentifier)' == '.NETCoreApp' and '$(_TargetFrameworkVersionWithoutV)' > '3.0') or
+                   ('$(TargetFrameworkIdentifier)' == '.NETStandard' and '$(_TargetFrameworkVersionWithoutV)' > '2.1') )">true</GenerateResourceUsePreserializedResources>
+    <GenerateResourceUsePreserializedResources>true</GenerateResourceUsePreserializedResources>
```

Comment:
Won't this just overwrite to true always?

Reply:
`timesDebuggingCodeLeakedIntoAPR++` 😳